shared/constants/fs.js

```diff
@@ -601,6 +595,17 @@ export const getTlfFromTlfs = (tlfs: Types.Tlfs, tlfType: Types.TlfType, name: s
   }
 }
 
+export const isOwnPublic = (p: Types.Path, username: string) => {
+  const parsedPath = parsePath(p)
+  switch (parsedPath.kind) {
+    case 'group-tlf':
+    case 'in-group-tlf':
+      return parsedPath.writers.contains(username)
+    default:
+      return false
+  }
+}
```

Comment:
IIRC "your own public TLF" means you're the only writer there, so just `.contains` wouldn't suffice.

But perhaps we should show it in all public TLFs? @cecileboucheron wdyt?

Reply:
I think we should show it in any public TLFs you have edit access to - hence `contains`. But interested to hear what Cécile thinks.

Reply:
Ah I see. In that case you could also use `writable` in the pathItem, which covers edge cases like when you access it through a social assertion handle as yourself.

That aside, it seems to me this would be a place to educate user about these stuff being open to the world, _and_ to let them know about keybase.pub, which is why I thought maybe we should show it for all. Design has final call though!

Reply:
Ah yeah good call, let's show it in all the public TLFs where you're a writer. So shared public folders too.